---
event_type: IssuesEvent
avatar: "https://avatars.githubusercontent.com/u/37929162?"
user: mergify[bot]
date: 2024-05-06
repo_name: flux-framework/flux-sched
html_url: https://github.com/flux-framework/flux-sched/issues/1172
repo_url: https://github.com/flux-framework/flux-sched
---

<a href='https://github.com/mergify[bot]' target='_blank'>mergify[bot]</a> closed issue <a href='https://github.com/flux-framework/flux-sched/issues/1172' target='_blank'>flux-framework/flux-sched#1172</a>.

<p>python/t10001-resourcegraph.py test fails outside of flux environment</p><small>Problem: when running `make check` on my Ubuntu desktop, this test fails for me, unless I run it in flux instance or via the `flux(1)` command front end....</small><a href='https://github.com/flux-framework/flux-sched/issues/1172' target='_blank'>View Comment</a>